// 网络通信协议格式.

package DataLock;

option java_package = "com.dlp.datalock";
option java_outer_classname = "MessagePackProtos";

// 消息包
message MessagePack {
  required string id  = 1;			// 消息ID
  required int32 type = 2;			// 消息类型    0: 为心跳
  required int32 cmd  = 3;			// 消息命令字
  required int32 len  = 4;			// 消息长度
  required bytes body = 5;			// 消息体
}

message ManagerNotifyMessage {
  required string ids = 1;
}